src/consumer/consumerGroup.js

```diff
@@ -182,13 +187,14 @@ module.exports = class ConsumerGroup {
     this.leaderId = groupData.leaderId
     this.memberId = groupData.memberId
     this.members = groupData.members
+    this.groupInstanceId = groupData.groupInstanceId
     this.groupProtocol = groupData.groupProtocol
   }
 
   async leave() {
-    const { groupId, memberId } = this
+    const { groupId, memberId, groupInstanceId } = this
     if (memberId) {
-      await this.coordinator.leaveGroup({ groupId, memberId })
+      await this.coordinator.leaveGroup({ groupId, memberId, groupInstanceId })
```

Comment:
We should not send leave group if static membership is enabled, i.e groupInstancId is defined.

Reply:
Can't you change it yourself? We're just waiting for this feature and no one is reacting!

Reply:
@abbccdda It is not necessary that anybody who uses static membership would never want to stop the graceful termination of consumption. This is a valid change. If one does not want to call leave group then just have to not call the consumer.stop() & consumer.disconnect() & leave group won't be called.

Reply:
Hey it seems like a lot of people are waiting for this (including me). I'd be willing to take this on since it seems like it's so close to being done. What's there left to do? Just this?